client/src/components/Sub_Questions/QuestionList.jsx

```diff
@@ -42,12 +46,26 @@ export default function QuestionList({ questionList, updateQuestions, productNam
       ))}
       {moreQuestions && (
         <button
-          className="stringbutton"
+          className="stringbutton bottom"
```

Comment:
I think classNames should be hyphenated right?

Reply:
@kevinleaves they are 2 different classes